### ISSUE TITLE
Documentation on dataset building

### ISSUE THREAD

**Issue by koopa31** [instanseg/instanseg#42]
Hi,

I've just created my own dataset.pth to train an InstanSeg model and actually it was not easy to find out which information, format file etc were necessary. 
I finally succeded in coding a script adapted to the data I had, but maybe that'd be nice to get a sort of documentation explaining how to build the pth file?

Thanks for this work!
Regards

**Comment by ThibautGoldsborough** [comment]
Really sorry to hear that. 

I suppose you had a look at the  "Add your own dataset [Optional]" cell in the load_datasets.ipynb? 

But yes, our documentation could be better.

**Comment by bobfRT1** [comment]
What would help me is a minimal example of the _masks.tiff for **combined** nucleus and cell training:
instanseg
    └───Raw_Datasets
        └─── Nucleus_Segmentation
           └───My_Own_Dataset
               └───img1.tiff
                   img1_masks.tiff
Specifically how both masks should look like: Areas with integers (1..n?) for  masks. How to annotate the background (0?) and demark "not annotated region" (-1)?

**Comment by ThibautGoldsborough** [comment]
> Specifically how both masks should look like: Areas with integers (1..n?) for masks. How to annotate the background (0?) and demark "not annotated region" (-1)?

Yes that's right. And they don't have to be matched (i.e nucleus 1 doesn't have to correspond to cell 1). 

You're expected to modify the code in the "load_custom_dataset" function.  Just uncomment the item["cell_masks"] and give it the cell_masks that you have.

**Comment by ThibautGoldsborough** [comment]
If you have -1 in your annotations you should delete the lines:

```
        masks, _ = fastremap.renumber(masks, in_place=True)
        masks = fastremap.refit(masks)
 ```
 
 in load_custom_dataset
